terraform-random-pet/variables.tf

```diff
@@ -2,17 +2,23 @@
 # The variables.tf file defines input variables that control the module’s configuration.
 
 # Best Practices:
-# - Descriptive Variables: Use meaningful names and description attributes.
-# - Default Values: Provide reasonable defaults when possible. For mandatory inputs, omit defaults to enforce explicit user input.
-# - Type Constraints and Validation: Use type constraints and validation blocks to catch incorrect inputs early.
-# - Group Related Variables: Organize variables logically, adding comments to separate sections if many variables exist.
+# - Descriptive variables: Use meaningful names and description attributes.
+# - Default values: Provide reasonable defaults when possible. For mandatory inputs, omit defaults to enforce explicit user input.
+# - Type constraints and validation: Use type constraints and validation blocks to catch incorrect inputs early.
+# - Group related variables: Organize variables logically, adding comments to separate sections if many variables exist.
```

Comment:
Let's delete this area of `variables.tf` since it duplicates info in the README

Reply:
Oops, I didn't save the file before commit+push.